CRM/Core/BAO/CustomField.php

```diff
@@ -153,42 +153,31 @@ public static function dataToHtml() {
    */
   public static function create($params) {
     $transaction = new CRM_Core_Transaction();
-    $origParams = array_merge([], $params);
     $params = self::prepareCreate($params);
 
     $customField = new CRM_Core_DAO_CustomField();
     $customField->copyValues($params);
     $customField->save();
 
-    // make sure all values are present in the object for further processing
-    $customField->find(TRUE);
```

Comment:
I remember there was some reason we had to add this...

Reply:
It's still there - just moved